testbin/workload/CiscoVT2people_320x192_12fps.yuv

```diff
@@ -0,0 +1 @@
uuux����������������������~}qqqqqqqqppppqqqqtrv|�{{~���������~���~|yvttttssssttttssssttttttttttttttttttttvvvvvvvvvvvvwwwwwwwwwwwwwwwwuwxyxxxxwvuswwwwvvvvvvvvvvvy�}}��������������������~qqqqqqqqooooqqqqtru||������������������{yttttrrrrttttrrrrttttttttttttttttttttuuuuuuuuuuuuwwwwwwwwwwwwxxxxuwxyyyyy~}{ywwwwuuuuuuuuuuux�|}��������������������~oooopppprrrroqrsusw}}}�������������������|yxsrtttttttttttttttttttttttttttttttttvvvvwwwwvvvvyxwuwwwwzywuy{|}zzzz��~}|yvwwwwvvvvvvvz�~���������������������llllmmmmoooooqrsusw}}���������������������~|xsqtttttttttttttttttttttttttttttttttuuuuwwwwuuuuyxwuwwwwuwxyz|}~}}}}~~~~��~}�{ywwwwuuuurux}�}���������������������lkihjjjjjjjjmnpqsuy|��������������������z~xsrtssssttttttttttttttttttttttttvvvvvvvvvvvvvvvvvvvvvvvvxz{|}}}}�������~}xvyxwutvsw~~����������������������~onlkiiiiiiiijlmnsuy{~���������������������}�xsqtrrrrttttttttttttttttttttttttuuuuuuuuuuuuuuuuuuuuuuuu{}~~~~~~~~~}}}}}��~���}xvyxwutvsx}~����������������������~ponlmmmmpppppppptrv||�����������������������vutrssssssssttttttttttttvvvvvvvvvvvvvvvvvvvvttttwwwwuwxy}��}�������~{}~�����|wxvvvvusy�������������������������ponloooooooorrrrtru||���������������������}vutrrrrrrrrrttttttttttttuuuuuuuuuuuuuuuuuuuuwwwwwwwwxz{|}��}�������~{}~����������~wuuuuus{�������������������������qqqqppppqqqqssssxqu��������~���������������~xsruttttttttssssttttttttvvvvvvvvvvvvwwwwwwwwwwwwyyyyy{�}~�������������~�����������zwwwwus{�}�����������������������qqqqooooqqqqrrrrtuy|�����������������������~{vtwttttttttuuuuttttttttuuuuuuuuuuuuwwwwwwwwwwwwxxxxy{���������������������������~wwwwut{�}�����������������������ppppppppqqqqqrtur��������������������������xvutrttttttttssssttttvvvvvvvvttttvvvvvvvvvvvvwwwwwwww{}~��������������}�����������yxwvsx}�����������������������ooooooooqqqqqrtu{��������������������������{vutrttttttttuuuuttttuuuuuuuuwwwwuuuuuuuuuuuuwwwwwwww{}~�����������������������������}xusy}}�����������������������rrrrqqqqqqqqssssx���������������������������zusvvvvvttttttttvvvvvvvvvvvvvvvvzywvvvvvvvvvwwwwwxz{�����������������������������}xtuy||������������������������ooooqqqqqqqqrrrrs}��������������������������zusvssssttttttttuuuuuuuuuuuuuuuuwvtsuuuuuuuuwwwwwxz{~~~~������������������������������{wuw|{������������������������ponlppppqqqqqqqqu{��������������������������vutrvvvvttttvvvvvvvvttttvvvvvvvvvvvvwwwwvvvvwwwwuw{}��������������������������}}�ysy�{������������������������ponlooooqqqqttttvy�������������������������}vutruuuuttttuuuuuuuuwwwwuuuuuuuuuuuuwwwwxxxxwwwwxz}�~~~~�������������������������������xsy~z������������������������ppppppppssssqrtuut{�����������������������~ttttvvvvvvvvvvvvvvvvvvvvwwwwvvvvvvvvvvvvwwwwwwwwz{}~������������������������������|ux|}������������������������oooooooorrrrqrtuuty������������������������yttttuuuuuuuuuuuuuuuuuuuuwwwwuuuuuuuuuuuuwwwwwwwwz{}~~~~~������������������������������|ux{~������������������������rnnrppppqqqqsssssuy|����������������������}uvvvvvvvvvvvvvvvvvvvvxxxxvvvvvvvvvvvvvvvvvvvvyyyy
```

Comment:
What is this for?

Reply:
This is for test in browser environment. All the files needed in test should be put under `testbin/workload`

Maybe I need to delete all the files under `testbin/workload` ?